lib/ws-channel.js

```diff
@@ -205,6 +204,12 @@ WebsocketChannel.prototype._attach = function(websocket) {
     websocket.on('open', this._onOpen);
   else
     this._onOpen();
+
+  // Clients flush when they recv SHK, servers flush after they send SHK.
+  if (!this._uri)
+    this._flushSendQueue();
+  // FIXME commenting out the flush above causes a packet sequence error
+  // that I also see ephemerally with the reconnect test.
```

Comment:
I found a bug when looking at https://github.com/strongloop/strong-control-channel/blob/master/test/test-ws-reconnect.js#L14, but it only happened with lots of fast packets, and I hadn't worked through it yet.

I can trigger the same exception with test-ws-server-flush if I remove the flushing of the send queue, so that should make it easier to figure out what's going on, but not tonight, its too late.